docs/advanced-usage.md

```diff
@@ -4,16 +4,8 @@ FakeItEasy exposes a few APIs that aren't commonly needed, but can be useful in
 
 ## Clearing the configuration of a fake
 
-The `Fake.ClearConfiguration` method removes all configured rules from a fake, leaving it in the default configuration.
-
-```csharp
-var foo = A.Fake<IFoo>();
-Assert.Equal(0, foo.Bar());
-A.CallTo(() => foo.Bar()).Returns(42);
-Assert.Equal(42, foo.Bar());
-Fake.ClearConfiguration(foo);
-Assert.Equal(0, foo.Bar());
-```
+The best way to remove a fake's configuration is to discard the fake and create
+a new one to replace it.

```

Comment:
Saying that it's the "best way" suggests there's an other way. Maybe you could mention that `ClearConfiguration` exists, but is deprecated? Or just don't say that it's the best way, and don't mention `ClearConfiguration`?

Reply:
You're right. I've given it another go (I just amended the commit; no changes to any other files). I'm keen to hear your thoughts.

Reply:
Perfect!